source/install/installing-ubuntu-2004-LTS.rst

```diff
@@ -20,18 +20,118 @@ Install a production-ready Mattermost system on up to three machines.
 .. include:: install-common-intro.rst
   :start-after: :nosearch:
 
-.. contents:: Install and configure the components in the following order. Note that you need only one database, either MySQL or PostgreSQL.
-  :backlinks: top
-  :local:
+.. contents:: Install and configure the components in the following order. 
+  :depth: 2
 
 .. include:: install-ubuntu-2004-server.rst
   :start-after: :nosearch:
 
-.. include:: install-ubuntu-2004-mysql.rst
-  :start-after: :nosearch:
+Install a database
+------------------
 
-.. include:: install-ubuntu-2004-postgresql.rst
-  :start-after: :nosearch:
+.. note:: 
+  
+  You only need one database: either MySQL or PostgreSQL. See the `database software <https://docs.mattermost.com/install/software-hardware-requirements.html#database-software>`__ documentation for details on database version support.
+
+.. tabs::
+
+  .. tab:: Install MySQL
+    
+    Install and set up a MySQL database for use by the Mattermost server.
+
+    1. Log into the server that will host the database, and open a terminal window.
+
+    2. Install MySQL.
+
+    3. Run ``sudo mysql_secure_installation`` and follow the instructions.
```

Comment:
What is this command?

Reply:
@nab-77 - Any ideas?

Reply:
Its a shell script to help admins secure their mysql install. Not sure we need to carry this level of detail here. Just need to ensure we call out the user and access rights we need.